src/pages/Categories.jsx

```diff
@@ -1,3 +1,31 @@
+import { Grid, GridItem } from '@chakra-ui/react';
+
+import { AccordionComponent, AccordionHeadings } from '@/components';
+
 export const Categories = () => {
-  return <></>;
+  return (
+    <>
+      <Grid templateColumns="repeat(2, 1fr)" height="100vh">
+        <GridItem className="expenseCol">
+          <AccordionHeadings
+            headingOne="Categories of Expenses "
+            headingTwo="Add expense Category"
+          />
+          <AccordionComponent name="Category 1" message="I am random message" />
+          <AccordionComponent name="Category 1" message="I am random message" />
+          <AccordionComponent name="Category 1" message="I am random message" />
+        </GridItem>
```

Comment:
maybe it's better to create like array maping to create categories components?

Reply:
Yea 😅. Funny, i did ask about arrays today. You're right, I'll implement that